### ISSUE TITLE
Hard coded time out in linux connect does not fit every need

### ISSUE THREAD

**Issue by Snagnar** [friedrith/node-wifi#88]
I'm trying to use your great module to connect to an AP launched by an ESP8266. Those tiny chips are not the fastest so connecting to them might take more than the hard coded 10 seconds used in linux-connect.js. Could you please make that configureable through options-object in the parameters? 

Thanks!

## Affected features

- [ ] node API
- [ ] cli

## Steps to Reproduce (for bugs)

1. 2. 3. 4.

## Your Environment

- OS:
- node-wifi version
- wifi card:

**Comment by friedrith** [comment]
Yes this is a good idea. Feel free to create a pull request for this feature.
